lupa/_lupa.pyx

```diff
@@ -102,6 +102,11 @@ class LuaSyntaxError(LuaError):
     """
 
 
+class LuaMemoryError(LuaError):
+    """Memory error in Lua code.
+    """
```

Comment:
To help distinguish Python and Lua memory errors better?

Reply:
> Is that important?

Clarity of the nature of an error is arguably important for some.

Reply:
But I understand that the memory space of Python and Lua are not much different.

Reply:
Especially since, under tight memory conditions, even allocating the `LuaMemoryError` instance may fail, in which case you would get a (pre-allocated) Python `MemoryError` instead. :)